client/components/container/AuthenticateUser.jsx

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import Authentication from '../presentation/Authentication/Index.jsx';
+import { signUpUser, signInUser } from '../../actions/actionCreators/UserActions';
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    onSignUpSubmit: ({ username, password, email, phoneNo }) => {
+      dispatch(signUpUser(username, password, email, phoneNo));
+    },
+    onSignInSubmit: ({ username, password }) => {
+      dispatch(signInUser(username, password));
+    }
+  };
+};
+
+const AuthenticateUser = connect(
```

Comment:
I have changed the mapDispatchToProps argument from a function to an object;
`{ SignInUser, SignUpUser }`